Makefile

```diff
@@ -92,3 +102,6 @@ gh-pages:
 .PHONY: api
 api:
 	oapi-codegen -config api/config.yaml api/arc.yml > api/arc.go
+
+.PHONY: clean_test
```

Comment:
clean test sounds like to clean up leftover from a test.
Maybe rename to something like: `clean_restart_e2e_test`